pkg/rbac/clusterrole.go

```diff
@@ -232,7 +241,7 @@ func (builder *ClusterRoleBuilder) Exists() bool {
 // validate will check that the builder and builder definition are properly initialized before
 // accessing any member fields.
 func (builder *ClusterRoleBuilder) validate() (bool, error) {
-	resourceCRD := "ClusterRole"
+	resourceCRD := "clusterRole"
```

Comment:
Why? new format?
I see plenty of examples where it's capitalized in the repo.

Reply:
Yes and this is an issue. We print it in the begging of the error message. In go all error has to start from small letter. Capitalized letters has to be avoided.